plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/AnalyticsPingManager.java

```diff
@@ -97,6 +102,14 @@ protected IStatus run(IProgressMonitor monitor) {
     }
   };
 
+  // Randomly create one when the plugin is activated? We use that one through
+  // the lifetime of plugin's process. If the user closes the IDE and starts a
+  // new one, our plugin will create a different UUID. That's OK though since 
+  // that's a different "session".
+  private final String uuid = UUID.randomUUID().toString();
```

Comment:
We can probably use `AnalyticsPingManager.getAnonymizedClientId()` for this?

Reply:
That tracks across restarts, which we don't need to do and thus shouldn't be doing. Better to reset every time.

Reply:
I thought this was a question about how we can track a unique user, whether regarding a user restarting IDE as a different user or trying to identify the single IDE workspace as one user. @etanshaul how do you go about this in KubeCode? Do we need to mimic the web session behavior as best as possible?

Reply:
http://go/gh/GoogleCloudPlatform/google-cloud-eclipse/pull/3405#pullrequestreview-233496480

I agree that this is about what we want to track, and I think `getAnonymizedClientId()` is better for our purpose.

Reply:
done